BoxContentSDK/BoxContentSDK/Requests/BOXRequest.m

```diff
@@ -80,6 +81,9 @@ - (NSURLRequest *)urlRequest
 
 - (void)performRequest
 {
+    if ([self.queueManager.session isKindOfClass:[BOXAppUserSession class]] && self.queueManager.delegate == nil) {
```

Comment:
Will do. Makes more sense since I'm accessing queueManager properties but for some reason I'm doing the asserting in BOXRequest....